src/style.css

```diff
@@ -1 +1,86 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+  height: 40px;
+}
+
+.nav {
+  box-sizing: border-box;
+}
+
+.logo__img {
+  align-self: center;
+}
```

Comment:
`align-self` works for items whose parent has `display: flex` or `display: grid`. this element's parent has neither of those, so it's not needed

Reply:
Ok